src/index.njk

```diff
@@ -8,16 +8,19 @@ is_generic: true
   <div class="bg-slate-300 dark:bg-slate-900 py-4">
     <div class="max-w-6xl mx-auto p-3">
       <h2 class="text-gray-900 dark:text-gray-100 text-4xl lg:text-5xl font-bold leading-normal text-center">
-        Quick bites of AWS <span class="text-primary-600 dark:text-secondary-400">every week!</span>
+        Insights and opinions about <span class="text-primary-600 dark:text-secondary-400">AWS</span> and <span class="text-primary-600 dark:text-secondary-400">the cloud</span>
       </h2>
-      <hr class="w-8 my-2 border-2 bg-secondary-500 border-secondary-500 dark:bg-primary-100 dark:border-primary-100 text-center mx-auto"/>
+      <hr class="w-8 mt-6 border-2 bg-secondary-500 border-secondary-500 dark:bg-primary-100 dark:border-primary-100 text-center mx-auto"/>
     </div>
     <p class="my-2 px-4 prose prose-lg md:prose-xl lg:prose-2xl dark:prose-dark mx-auto">
-      AWS Bites is a weekly <strong>podcast</strong> where we answer interesting questions about <strong>AWS</strong> and <strong>cloud computing</strong>.
-      A new episode goes live <strong>every Friday morning</strong>.
-      Brought to you by <a href="https://twitter.com/eoins">@eoins</a> and <a href="https://twitter.com/loige">@loige</a>.
+      AWS Bites is the <strong>podcast</strong> for <strong>cloud consultants</strong> and <strong>practitioners</strong>, hosted by <a href="https://twitter.com/eoins">Eoin Shanaghy</a> and <a href="https://twitter.com/loige">Luciano Mammino</a>.
```

Comment:
Is it for "cloud consultants" or _by_ cloud consultants? 😃 Maybe we can leave it as "cloud practitioners"?